LICENSE

```diff
@@ -9,13 +9,19 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
 furnished to do so, subject to the following conditions:
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+the above copyright notice and this permission notice shall be included in all
+  copies or substantial portions of the software.
+  the software is provided "AS IS",
+  without warranty of any kind,
+  express or
+  implied,
+  including but not limited to the warranties of merchantability,
+  fitness for a particular purpose and noninfringement. in no event shall the
+  authors or copyright holders be liable for any claim,
+  damages or other
+  liability,
+  whether in an action of contract,
+  tort or otherwise,
+  arising from,
+  out of or in connection with the software or the use or other dealings in the
+  software.;LicenseCopyright2023QaisarAbbasPermissionisherebygranted,freeofcharge,toanypersonobtainingacopyofthissoftwareandassociateddocumentationfilesthetodealintheSoftwarewithoutrestriction,includingwithoutlimitationtherightstouse,copy,modify,merge,publish,distribute,sublicense,and/orsellcopiesoftheSoftware,andtopermitpersonstowhomtheSoftwareisfurnishedtodoso,subjecttothefollowingconditions
```

Comment:
- [ ] Please format your `LICENSE` file correctly, it seems you tried to modify the LICENSE content and because of some markdown, your license content is distorted and it is really difficult to read the terms and conditions of the license for thee usage of this project. Please fix the formatting issue.